pyrogram/client/client.py

```diff
@@ -2279,7 +2301,18 @@ def get_file(self,
                 cdn_session.start()
 
                 try:
-                    with open(file_name, "wb") as f:
+                    # cant fdopen the closed file descriptor which could be closed due
```

Comment:
Code flow will follow either one of these two branches, doing so there will be a zombie fd in case a file is stored in CDNs.

Reply:
I actually tested your code above before reading you were actually removing it, but I saw some extra unused files in the temp directory (Windows) and that's weird

Reply:
Ok, the reason is that you did not close the fd, and we get an OSError which will be swallowed by the try catch below.

Reply:
Yes, this is what I was thinking when you mentioned that only one of those branches is going to be hit.

The first call to **tempfile.mkstemp** here:

https://github.com/EriHoss/pyrogram/blob/8796e857af44918a05cf37fb3b5f42cbca6fb7c4/pyrogram/client/client.py#L2264

Can be moved into the first branch above where the file descriptor is opened in the **with** statement to cure this.

And this logic can be removed:

https://github.com/EriHoss/pyrogram/blob/8796e857af44918a05cf37fb3b5f42cbca6fb7c4/pyrogram/client/client.py#L2308

I'll go ahead and change this.